docs/conf.py

```diff
@@ -111,47 +111,52 @@
         },
         {
             'name':
-            'Projects',
+            'Docs',
```

Comment:
`Docs` is a little confusing. The first impression of users should be "it's some other documents of MMCV", but they get a list of repos.

Reply:
This is not a problem, the user will know what the docs are when they click on them.